addons/stock_account/models/product.py

```diff
@@ -114,11 +109,6 @@ class ProductProduct(models.Model):
     stock_value = fields.Float(
         'Value', compute='_compute_stock_value')
 
-    @api.onchange('type')
-    def onchange_type_valuation(self):
-        # TO REMOVE IN MASTER
-        pass
-
     @api.multi
     def do_change_standard_price(self, new_price, account_id):
         """ Changes the Standard Price of Product and creates an account move accordingly."""
```

Comment:
There are location for which this product could not have any qty_available,
`for location in locations:`

It would be better if this is done by first checking which locations actually
have this product.

https://github.com/Vauxoo/addons-vauxoo/blob/8.0/product_extended_variants/model/template.py#L101-L113

@moylop260 @nhomar Do you remember this performance problem?